ui/src/app/components/help-sidebar.spec.tsx

```diff
@@ -136,34 +128,82 @@ describe('HelpSidebar', () => {
   let runtimeStub: RuntimeApiStub;
   let appsStub: AppsApiStub;
   let props: {};
+  let user;
 
   const component = async () => {
-    const c = mountWithRouter(
-      <SWRConfig value={{ provider: () => new Map() }}>
+    /*
+    The useSWR hook in useGenomicExtractionJobs is causing
+    extractionJobs to carryover between tests. More details can be
+    found here:
+    https://github.com/vercel/swr/issues/781#issuecomment-952738214
+     */
+    const c = renderWithRouter(
+      <SWRConfig value={{ provider: () => new Map(), dedupingInterval: 0 }}>
         <HelpSidebar {...props} />
-      </SWRConfig>,
-      {
-        attachTo: document.getElementById('root'),
-      }
+      </SWRConfig>
     );
-    await waitForFakeTimersAndUpdate(c);
     return c;
   };
 
-  const runtimeStatusIcon = (wrapper, exists = true) => {
-    const icon = wrapper
-      .find({ 'data-test-id': 'runtime-status-icon-container' })
-      .find('svg');
-    expect(icon.exists()).toEqual(exists);
-    return icon;
+  const findRuntimeStatusIcon = async (status: RuntimeStatus) => {
+    let label: RegExp;
+    switch (status) {
```

Comment:
I typically prefer `switchCase()` when the action to take is variable assignment